Cargo.toml

```diff
@@ -26,7 +26,8 @@ default-target = "x86_64-pc-windows-msvc"
 
 [dependencies]
 libloading = "0.7.0"
-com-rs = "0.2.1"
+# https://github.com/microsoft/com-rs/pull/234
+com = { git = "https://github.com/MarijnS95/com-rs", rev = "c8256cb", features = ["production"] }
```

Comment:
don't think we should merge and publish with these :(

Reply:
And we don't need it, published `com 0.6` works for us now that https://github.com/microsoft/DirectXShaderCompiler/pull/3793 has been merged upstream and we don't need any vtable hacks behind `cfg()` anymore :tada: